rep-0003.rst

```diff
@@ -180,6 +180,49 @@ Buildsystem support:
 
   - build from source
 
+Kinetic Kame (May 2016 - May 2018)
```

Comment:
Same end date as Ubuntu X (i.e. 2021) or the release date of the next ROS distro that supports a newer Ubuntu LTS? (i.e. 2018)